packages/host/tests/acceptance/code-submode/spec-test.gts

```diff
@@ -568,6 +630,42 @@ module('Acceptance | Spec preview', function (hooks) {
       .exists();
   });
 
+  test('navigation: spec preview updates when changing between different declarations inside inspector', async function (assert) {
```

Comment:
this test checks for navigation by lhs. But also, deals with if you interacted with the dropdown